<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgLocalRecord</class>
 <widget class="QDialog" name="DlgLocalRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>本地录制/推流</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QComboBox, QPushButton
{
	min-height: 30px;
	min-width: 100px;
	border: 1px solid rgb(221,221,221);
	border-radius: 3px;
}

QPushButton:hover{
	background: rgb(239,239,239);
}

QPushButton:pressed{
	background: rgb(228,228,228);
}

QCheckBox{
	min-height: 30px;
}

#DlgLocalRecord {
	background: rgb(255,255,255);
}

QComboBox {
	padding-left: 8px;
	padding-right: 8px;
}
QComboBox:hover {
	border-color:rgb(57,129,252);
}
QComboBox::drop-down {
	padding-top: 5px;
	image: url(:/Resources/dropDown.png);
}
QComboBox QAbstractItemView{
	margin-top:1;
	outline:0;	
	border:1px solid rgb(225,225,225);
	background-color:rgb(255,255,255);
	selection-background-color:rgb(80,145,255);
}
QComboBox QAbstractItemView::item{
	min-height:30px;
	max-height:30px;
}
QComboBox QAbstractItemView::item:hover{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
QComboBox QAbstractItemView::item:selected{
    background-color:rgb(240,240,240);
	color:rgb(45,45,45);
}
#btnStartRecord, #btnStopRecord {
	background:rgb(57,129,252);
	color: rgb(255,255,255);
	border: 1px solid rgb(57,129,252);
	min-width: 100px;
}
#btnStartRecord:hover,#btnStopRecord:hover{
	background: rgb(80,145,255);
}

#btnStartRecord:pressed,#btnStopRecord:pressed{
	background: rgb(43,121,255);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>清晰度：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbDefinition">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>标清</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>高清</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>超清</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>格式：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbFormat">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>mp4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>flv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>avi</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ckOutputFile">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>输出到文件：</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="outputFileWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="editPath">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnChoosePath">
             <property name="text">
              <string>选择存储目录</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btnRecordPlay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>录像回放</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="markPage" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="editMarkDesc">
                <property name="text">
                 <string>自定义打点描述</string>
                </property>
                <property name="placeholderText">
                 <string>自定义打点描述</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnMark">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>新增打点</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="ckOutputUrl">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>输出到url:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editUrl">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>支持的推流协议：rtmp、rtsp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnStartRecord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopRecord">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
